frontend/src/components/VAudioTrack/VGlobalAudioTrack.vue

```diff
@@ -184,10 +184,8 @@ export default defineComponent({
     const pause = () => activeAudio.obj.value?.pause()
 
     /* Timekeeping */
-    const message = computed<string | undefined>(() =>
-      activeMediaStore.message
-        ? i18n.t(`audio-track.messages.${activeMediaStore.message}`).toString()
-        : undefined
+    const message = computed<string | undefined>(
+      () => activeMediaStore.message ?? undefined
```

Comment:
`activeMediaStore` saves message already translated, so we don't need to use i18n here